packages/outputarea/src/model.ts

```diff
@@ -461,15 +460,15 @@ class OutputAreaModel implements IOutputAreaModel {
   /**
    * Handle a change to an item.
    */
-  private _onGenericChange(): void {
-    this._stateChanged.emit(void 0);
+  private _onGenericChange(sender: IOutputModel, model: IOutputModel): void {
+    this._stateChanged.emit(model);
```

Comment:
This could simply emit the `sender` instead of the `model`, right? Then you could avoid the changes to `OutputModel`.